builtin/providers/google/resource_compute_instance_template.go

```diff
@@ -305,11 +305,9 @@ func buildNetworks(d *schema.ResourceData, meta interface{}) (error, []*compute.
 	for i := 0; i < networksCount; i++ {
 		prefix := fmt.Sprintf("network_interface.%d", i)
 
-		source := "global/networks/default"
+		source := "global/networks/"
```

Comment:
I didn't realize global/networks/default would actually be accepted by the API (usually it takes just "default").  So in that case, what you've got here is fine.